string tst_file # Name of the TST file to execute
---
bool success # Whether the execution was successful
string error_message # Error message if the execution was unsuccessful
